UpcomingMovies/UpcomingMovies/Component/MovieComponent.cs

```diff
@@ -0,0 +1,111 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using UpcomingMovies.Consts;
+using UpcomingMovies.DTO;
+using UpcomingMovies.Enums;
+using UpcomingMovies.Model;
+using UpcomingMovies.Parameter;
+using UpcomingMovies.Service;
+
+namespace UpcomingMovies.Component
+{
+    public class MovieComponent
+    {
+        IService _baseService;
+        public MovieComponent()
+        {
+
+        }
+        public MovieComponent(IService service)
```

Comment:
This constructor doesn't look to be used.

Reply:
I was going to mock this class it but I forgot to remove this one.